internal/backend/sessionsgrpcsvc/svc.go

```diff
@@ -71,6 +71,25 @@ func (s *server) Get(ctx context.Context, req *connect.Request[sessionsv1.GetReq
 	return connect.NewResponse(&sessionsv1.GetResponse{Session: session.ToProto()}), nil
 }
 
+func (s *server) Stop(ctx context.Context, req *connect.Request[sessionsv1.StopRequest]) (*connect.Response[sessionsv1.StopResponse], error) {
```

Comment:
how stopped sesstions are handled in DB?

Reply:
not sure i follow your question here

Reply:
I meant that there are db functions that should be avare of this change, e.g. listDeploymentsWithStats.
another things - there maybe places where only finished and error states considered as final session states. After your change they should be aware that STOPPED is also a final state

Reply:
ah, i think that is the only one? fixed.